couchbase/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "2"
 services:
   db:
-    image: arungupta/couchbase
+    image: CelsoSantos/couchbase
```

Comment:
It's not. I don't recall including that on the PR version, but it should stay with arungupta/couchbase